src/views/product-tutorials-view/helpers/get-inline-content.ts

```diff
@@ -6,11 +6,11 @@ import {
 } from 'lib/learn-client/types'
 
 export interface InlineTutorials {
-  [slug: string]: ClientTutorial[]
+  [slug: string]: ClientTutorial
```

Comment:
I _think_ this type was previously slightly off; I believe `InlineTutorials` & `InlineCollections` have values that are individual items, not arrays.

Reply:
Good catch